src/atwho.js

```diff
@@ -131,7 +132,25 @@ define([
                                                        options.property);
                             }
                             return value;
-                        }
+                        },
+                        afterMatchFailed: function(at, $el) {
+                            if (options.useRichText) {
+                                var content = $el.html().trim().replace(/^.*\s/, "");
+                                var isUnknownCaseHashtag = richText.CASE_REF_REGEX.test(content) && content.replace(richText.CASE_REF_REGEX, "");
+                                if (isUnknownCaseHashtag || form.isValidHashtag(content)) {
+                                    options.functionOverrides.insert.call(this, content);
+                                }
+                            }
+
+                            // Gracelessly force focus back to input...otherwise, later code steals
+                            // the focus, or, occasionally, gives focus to a different input.
+                            var $inputor = this.$inputor;
+                            _.delay(function() {
+                                $inputor.focus();
+                            }, 100);
```

Comment:
ah, you're right. Personally I tend to go with shorter if possible. That way if there are multiple elements causing delays you can stack a bunch of delays without going over 100ms.